generate-summary.py

```diff
@@ -41,14 +41,23 @@
 		i += 1
 
 # just copy values to jure.xls
+result = []
 for t in total:
 	cnt = sum(t[1])
 	if cnt == 0:
 		continue
 	if rel:
 		out=""
+		out2 = []
 		for c in t[1]:
 			out+= str(float(c*100) / float(cnt))[:4] + " " 
+			out2.append(round(((c*100)/cnt),1))
```

Comment:
Like before, this doesn't guarantee the shares to add up to 100%, but it's fine, we don't need to care about promiles.